addons/website_sale/static/src/scss/website_sale.scss

```
@@ -825,4 +825,8 @@ a.no-decoration {
 
 .accordion-button:not(.collapsed).bg-transparent::after {
     background-image: escape-svg($accordion-button-icon);
+}
+
+.object-fit-contain {
+    object-fit: contain !important;
 }
```

Don't forget Unix conventions on text files' structure based on a line definition (https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)
```suggestion
}

```